app/mci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -21,10 +21,10 @@ import (
 	"github.com/golang/glog"
 	"github.com/hashicorp/go-multierror"
 	compute "google.golang.org/api/compute/v1"
+	ingressbs "k8s.io/ingress-gce/pkg/backends"
```

Comment:
why ingressbs? Is that auto-selected? Seems like "ingressb" or "ingressbe" would be more inline with "ingresshc" below.

Reply:
Updated to ingressbe